### ISSUE TITLE
Wrong indentation

### ISSUE THREAD

**Issue by sassanh** [MaxMEllon/vim-jsx-pretty#105]
I'm thankful for this great plugin you made, I had my own fork of `vim-jsx` and used it all these years but I had a hard time updating it with new technologies (like typescript, etc) I'm really glad that this plugin works with typescript out of the box.

I have a minor issue with indentation, consider this block of code:

```jsx
<div
    style={{
        width: '100%',
    }}>
        <div/>
</div>;
```

This is how it's blocked by `vim-jsx-pretty` but I think the inner `div` and `span` should have 1 less indentation.

**Comment by yuezk** [comment]
Thank you for your feedback, I will take a look. I think the workaround for this is like below.


```jsx
<div
    style={{
        width: '100%',
    }}
>
    <div/>
</div>;
```

**Comment by yuezk** [comment]
@sassanh This issue should have been fixed in the latest version. Please update this plugin and verify it. Thanks.

**Comment by sassanh** [comment]
@yuezk it's working perfectly fine except one issue, I tried it with some big files I have, and it didn't change anything when I did a `gg=G` except ternary ifs, when I have something like this:

```javascript
<div>
    {condition
            ? <div/>
            : <span/>
    }
</div>
```

it adds an additional indentation level, I had long discussions with `vim-javascript` authors like this one: https://github.com/pangloss/vim-javascript/issues/618#issuecomment-243968811

**Comment by yuezk** [comment]
I see, I will keep improving indentation.

**Comment by yuezk** [comment]
Hi, @sassanh Fixed in the latest commit.

**Comment by sassanh** [comment]
It's working great now, I checked it with some big files. There's a weird behavior sometimes: if I press `gg=G` some lines sometimes get weird indentation, if I press `ugg=G` this time another set of files may get weird indentation, sometimes all indentation is correct, but in the same file if I run indentation again it may ruin some lines, if I go to the ruined lines and press `==` one by one it indents correctly. It may be related to one of my other plugins. I just wanted to share this observation with you in case it triggers something in your mind. If it doesn't, I'll see if it makes any troubles in future and will try a way to reproduce it and will create a new issue.

Other than that it's working perfectly fine and it satisfies all my concurrent requirements. Thanks!

**Comment by yuezk** [comment]
OK, feel free to open a new issue if you encountered a new issue. Thanks.
